tests/integration/components/conversations/conversation-part-closed-test.js

```diff
@@ -27,18 +27,42 @@ moduleForComponent('conversations/conversation-part-closed', 'Integration | Comp
   }
 });
 
-test('if current user closes message, "You closed this at" is rendered', function(assert) {
+test('if current user closes message, "You closed this" is rendered', function(assert) {
   assert.expect(1);
 
   let user = {
-    id: 1
+    id: 1,
+    username: 'testuser'
   };
 
+  let closedAt = 'MM/DD/YYYY hh:mm:ss';
+
   set(this, 'author', user);
-  set(this, 'closedAt', moment().subtract(2, 'days'));
+  set(this, 'closedAt', moment().fromNow());
```

Comment:
I found `fromNow()); from the moment docs here [](http://momentjs.com/docs/#/displaying/from/).